tests/test_icml_conference.py

```diff
@@ -1684,6 +1684,10 @@ def test_reviewer_reassingment(self, client, openreview_client, helpers, seleniu
         messages = openreview_client.get_messages(to='rachel_bis@icml.cc', subject='[ICML 2023] Reviewer Invitation declined for paper 1')
         assert len(messages) == 1
 
+        invite_edges=openreview_client.get_edges(invitation='ICML.cc/2023/Conference/Reviewers/-/Invite_Assignment', head=submissions[0].id, tail='~Rachel_ICML2')
+        assert len(invite_edges) == 1
+        assert invite_edges[0].label == 'Declined'
```

Comment:
I tested adding a comment in my local, but not sure how to add this to test

Reply:
I think with Selenium you can select the textarea and enter a comment. You can add a new parameter in `def respond_invitation(selenium, request_page, url, accept, quota=None):
` to pass a comment when accept is False. You can check in the implementation how we select a quota from the dropdown, you can do something similar with the textarea.